docs/reference/filebeat/filebeat-input-cel.md

```diff
@@ -294,6 +294,24 @@ filebeat.inputs:
   resource.url: http://localhost
 ```
 
+```yaml
+filebeat.inputs:
+- type: cel
+  auth.token:
+    type: Bearer
+    value: supersecret_bearer_token
+  resource.url: http://localhost
+```
+
+```yaml
+filebeat.inputs:
+- type: cel
+  auth.token:
+    type: Token
+    value: supersecret_token
+  resource.url: http://localhost
+```
+
```

Comment:
Maybe add some intro text that can be tagged with [inline `applies_to`](https://elastic.github.io/docs-builder/syntax/applies/#inline-annotations) to clarify in which versions these are supported (it might be helpful to add similar into text to the other code blocks above, too):

````suggestion

**Token authentication (Bearer)** {applies_to}`stack: ga 9.1.0`

```yaml
filebeat.inputs:
- type: cel
  auth.token:
    type: Bearer
    value: supersecret_bearer_token
  resource.url: http://localhost
```

**Token authentication (Token)** {applies_to}`stack: ga 9.1.0`

```yaml
filebeat.inputs:
- type: cel
  auth.token:
    type: Token
    value: supersecret_token
  resource.url: http://localhost
```

````